### ISSUE TITLE
Compose Labeling throws RuntimeException

### ISSUE THREAD

**Issue by imagejan** [knime-ip/knip#315]
When using the GroupBy node with Compose Labeling (using the nightly builds), I get the following exception:

```
ERROR GroupBy              0:166:35   Execute failed: java.lang.RuntimeException: java.lang.ClassCastException: net.imglib2.img.array.ArrayImg cannot be cast to org.knime.knip.base.data.img.ImgPlusCellMetadata
```

These are my settings:

![composelabeling](https://cloud.githubusercontent.com/assets/2033938/13636071/03c67d7c-e600-11e5-953f-f1c4a01b9712.png)

**Comment by hornm** [comment]
Unfortunately I can't reproduce the problem, Jan. Could you send me the full stack trace, or even better a minimal workflow?

**Comment by imagejan** [comment]
I'm unable to reproduce this with a smaller data set and a minimal workflow yet.

It happened when I wanted to group a relatively large table, and the exception was thrown at about 30% progress on the GroupBy node. I'm now trying to loop over the single groups to find out if there is one specific that causes this problem.

**Comment by imagejan** [comment]
I managed to find an image that causes the error.

Here's a workflow to reproduce:
[GroupBy_Labeling_Bug.zip](https://github.com/knime-ip/knip/files/165344/GroupBy_Labeling_Bug.zip)

Get the tif example image from [here](http://wwwapp.fmi.ch/fileexchange/GetFile.aspx?ID=32ff8e6c428643a0bece741f3b66bc27&Cfg=&file=1&method=standard).

**Comment by hornm** [comment]
Thanks, Jan! I fixed the problem with 472c8fc2a3e9c6dd4f7723c76df544ad7b9e1490. It is actually the "Feature Calculator (Beta)" that causes the problem.
